settings.gradle

```diff
@@ -0,0 +1,11 @@
+pluginManagement {
+    repositories {
+        if (gradle.startParameter.isOffline()) {
+            println "Using offline plugin repository"
+            maven {url uri(projectMavenRepo) }
+        } else {
+            println "Using online plugin repository"
+            gradlePluginPortal()
+        }
+    }
+}
```

Comment:
crucial bit to make saliman plugin work in offline mode..

Reply:
Also note it looks for startParameter.isOffline, not for disconnected property. Env properties not loaded yet at that stage, so you'd have to provide it from commandline (using -Pdisconnected=true), which kinda beats the purpose of using a property..